### PR TITLE
html document template contents and main page copy

### PR DESCRIPTION
**Pull request by LornaHarwood**
## Changes in this PR

Created the `Content` column
This is comprised of all the `h2` tag headers from the `body` text.

The `Related content` anchor and placeholder has been set up for when this functionality has been set up. The `Related content` functionality is out of scope for this work.

The `Show all updates` frontend is in place. The backend for this is out of scope for this work.

### Before
Screenshot 2022-04-27 at 14 52 24

### After
Screenshot 2022-04-27 at 14 51 47

### CODE REVIEW

File: cms/publications/templates/publications/publication.html [hunk 1]
```diff
@@ -70,8 +71,41 @@
...
+                    <a href="#related_contents">Related Content</a>
+                </nav>
+            </div>
+            <div class="nhsuk-grid-column-two-thirds nhsuk-u-margin-top-4">
+                <div id="content">{{ self.body|richtext }}</div>
+                <hr>
+                <div id="see-all-updates" class="nhsuk-review-date last-publication">
+                    <p class="nhsei-body-s">
+                        <strong>Date published:</strong> {{ self.first_published_at|date:'d M Y' }}<br>
+                        <strong>Date Last updated:</strong> {{ self.latest_revision_created_at|date:'d M Y' }}
```
Comment (dragon-dxw) [inline cms/publications/templates/publications/publication.html:88]: Bikesheddy: I feel like the first letters of 'last' and 'updated' should both be capitalised, or neither.

File: cms/publications/templates/publications/toc.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% if toc_items %}
   <div>
-    <h2>Contents</h2>
+    <h2>Content</h2>
```
Comment (dragon-dxw) [inline cms/publications/templates/publications/toc.html:3]: Even more bikesheddy: I'd say Contents was right as a header for the table of contents (as opposed to the technical term for 'this piece of text and images', where 'the content' makes sense)

**Review by dragon-dxw — APPROVED** [review]
These comments are incredibly nitpicky, sorry; please take that as a big thumbs up to the substance of the PR.

File: cms/publications/templates/publications/publication.html (hunk 1 above)
Comment (jacksonj04) [reply cms/publications/templates/publications/publication.html]: I think the style manual says sentence case is right here, "Date published" and "Date last updated".

**Comment by jacksonj04** [comment]
Is "related content" actually a menu item here, or is it just left ofter from testing data? I'm not sure it should be in the auto-generated TOC.

**Pull request merged**